### DISCUSSION TITLE
【Next主题】在宽度较小的屏幕上显示Next公告栏

### DISCUSSION THREAD

**Discussion by Ylarod** [tangly1024/NotionNext#2035] [Ideas]
**期望的正常结果**
宽度较窄时将公告移动到左下？

**截图**

<img width="1919" alt="iShot_2023-03-11_11 44 06" src="https://user-images.githubusercontent.com/30978685/224463209-a9019b6a-fa6d-4e0e-b82a-15c100605126.png">

MBP 14寸：

<img width="1511" alt="iShot_2023-03-11_11 44 19" src="https://user-images.githubusercontent.com/30978685/224463210-86d998b3-fbd4-4d44-9f00-ba2239cc0e85.png">

**Comment by tangly1024** [comment]
https://github.com/tangly1024/NotionNext/issues/929
